src/index.ts

```diff
@@ -1,7 +1,16 @@
-import getPool from "./getPool";
+import { makePool } from "./pool";
+import { queryAny, queryNone, queryOne, queryOneOrMore, queryOneOrNone } from "./query";
```

Comment:
`export { } from "./query";` ?

Reply:
I have an aversion to `export * from`. I like the specificity in a "root" `index.ts`.